### PR TITLE
Frontend Appearence

### PR DESCRIPTION
**Pull request by Tudor900**
smekerie

### CODE REVIEW

File: src/shared/components/styles/home.css [hunk 1]
```diff
@@ -2,122 +2,133 @@
...
-    background-size: 20px 20px;
-    background-position: center center;
-    box-shadow: 0 1px 24px 1px rgba(0, 0, 0,0.48);
-    border-radius: 50%;
-    max-width: 1500px;
-    max-height: 2000px;
-    min-width: 305px;
-    margin: auto;
-    margin-top: 100px;
+.soto1
```
Comment (Alex7734) [inline src/shared/components/styles/home.css:5]: please have a better naming xD

File: src/shared/components/styles/home.css [hunk 2]
```diff
@@ -2,122 +2,133 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 50px;
+    background: rgb(50,50,49);
+    background: linear-gradient(180deg, rgba(50,50,49,1) 0%, rgba(50,50,49,1) 80%, rgba(153,26,26,1) 100%);
+    
+}
+
+.soto1 {
```
Comment (Alex7734) [inline src/shared/components/styles/home.css:129]: here as well

File: src/shared/components/Home.tsx [hunk 3]
```diff
@@ -137,70 +137,31 @@ const Home = (props: any) => {
...
+        <div className='echipa1'>{game?.team1?.name}: {game?.team1?.points} </div>
+        <div className='puncteRunda'> {points} </div>
+        <div className='echipa2'>{game?.team2?.name}: {game?.team2?.points} </div>
       </div>
-      <Container maxWidth="lg" style={{ paddingTop: 30, paddingBottom: 50 }}>
-        <Box sx={{ flexGrow: 2 }}>
-          <div style={{ display: 'flex', justifyContent: 'center' }}>
-            <div className="scor" id={'boardScore'}>
-              <p style={{ fontSize: 25 }}>{points}</p>
+      <div className='intrebare'>{currentQuestion?.text}</div>
```
Comment (Alex7734) [inline src/shared/components/Home.tsx:146]: {game && game.questionRevealed
                ? currentQuestion?.text
                : 'Coming...'}

File: src/shared/components/styles/home.css [hunk 4]
```diff
@@ -2,122 +2,133 @@
...
+    font-size: 50px;
+
 }
 
-.questions {
+.tablaRaspunsuri {
+    background-color: #991a1a;
+    width: 1500px;
+    height: 635px;
+    border: 5px solid white; /* Combined border shorthand */
```
Comment (Alex7734) [inline src/shared/components/styles/home.css:82]: remove comm

File: src/shared/components/styles/home.css [hunk 5]
```diff
@@ -2,122 +2,133 @@
...
-    color: white;
-}
\ No newline at end of file
+.tablaRaspunsuri .raspunsuri {
+    margin: auto;
+    width: 700px;
+    height: 125px;
+    border: 5px solid white; /* You can adjust the border width as needed */
+    border-radius: 25px; /* Adjust the border-radius as needed */
+    /* Add content or specify dimensions to make the borders visible */
```
Comment (Alex7734) [inline src/shared/components/styles/home.css:119]: Remove comm from ChatGPT xD

File: src/shared/components/styles/home.css (hunk 1 above)
Comment (Tudor900) [reply src/shared/components/styles/home.css:5]: nu

**Pull request closed without merging**